setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore=E111,E114,E121,E126,E127,E128,E131
```

Comment:
Could we comment what these different codes are, just so that we're tracking what the style is?

Reply:
Thanks!